src/contour/Actions.h

```diff
@@ -136,6 +136,156 @@ using Action = std::variant<CancelSelection,
 
 std::optional<Action> fromString(std::string const& name);
 
+namespace documentation
+{
+
+    constexpr std::string_view CancelSelection { "Cancels currently active selection, if any." };
```

Comment:
```suggestion
    constexpr inline std::string_view CancelSelection { "Cancels currently active selection, if any." };
```

Don't you think they should all be marked `inline`, because they're in the header? 🤔